src/swapi_client/v2/models/base.py

```
@@ -0,0 +1,199 @@
+        payload = {"data": self._dirty}
+        resp = await self.client.patch(f"{self.endpoint}/{self.pk}", json=payload)
+
+        data = resp.get("data") or resp
+        return self._reload_from_data(data)
+
+    # ---------------------------------------------------------
+    # DELETE
+    # ---------------------------------------------------------
+    async def delete(self) -> None:
+        if self.pk is None:
+            raise SWAPIError("Cannot delete object without primary key")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)